content/chainguard/chainguard-images/features/ca-docs/custom-assembly.md

```diff
@@ -43,67 +45,28 @@ The packages you can add to a container image are those that your organization a
 
 The changes you make to your customized container image may affect its functional behavior when deployed. Chainguard doesn’t test your final customized image and therefore doesn't guarantee its functional behavior. Please test your customized images extensively to ensure they meet your requirements.
 
-Lastly, while Custom Assembly is in its beta phase it can only be configured from the Chainguard Console.
+### Chainguard's CVE SLA and Custom Assembly
 
+Today, Chainguard delivers an [SLA for CVE remediation](https://www.chainguard.dev/legal/cve-sla) for all of its container images. However, when you re-configure Chainguard’s standard, off-the-shelf images, this SLA doesn't extend to packages you add on because in this scenario Chainguard doesn't have any control over the build. This means that you would be responsible for maintaining these added packages or run the risk of accruing CVEs.
```

Comment:
This doesn't sound quite right. We're building the packages, why don't we have control? How can the customer "be responsible"? What options do they have to fix a CVE? Reading the next para, I wonder if you meant to delete this?

Reply:
Left a suggestion on slack, but I'll approve assuming this wording has already been discussed and vetted.